internal/testutil/assert/assert.go

```diff
@@ -73,12 +73,48 @@ func ErrorMatchesf(tb miniTB, pattern string, err error, msgFormat string, args
 	}
 }
 
-func ErrorIs(tb miniTB, want, got error) {
+func ErrorIs(tb miniTB, got, target error) {
```

Comment:
I prefer `want` to `target` here.

Reply:
I used the errors.Is idiomatic arguments

https://pkg.go.dev/errors#Is

But I can change for sure

Reply:
I now use `got, want`

Reply:
thanks